ClassicAssist/UI/ViewModels/Agents/ScavengerTabViewModel.cs

```diff
@@ -183,10 +191,12 @@ private void ItemsOnCollectionChanged( int totalcount, bool added, Item[] items
 
         internal void CheckArea()
         {
-            if ( !Enabled || Engine.Player == null )
+            //BEGIN
+            if (!Enabled || Engine.Player == null || Engine.Player.IsDead)
```

Comment:
Fix the formatting... If ( !Enabled ) spaces between parentheses
Remove comments

Reply:
Done